engine-comparison/begin-experiment.sh

```diff
@@ -34,28 +34,12 @@ done
 gcloud compute scp fengine-configs/ ${INSTANCE_NAME}:~/input --recurse
 rm -r fengine-configs
 
-# These will frequently/usually be defined by the user
-JOBS=${JOBS:-8}
-N_ITERATIONS=${N_ITERATIONS:-5}
+CONFIG=${SCRIPT_DIR}/config
+echo "BMARKS=$1" > ${CONFIG}/bmarks.cfg
```

Comment:
Maybe combine bmarks.cfg into parameters.cfg?  Then the user could specify everything in that file and be good to go.